cdrom/src/lib.rs

```diff
@@ -435,7 +435,10 @@ impl Sector {
     //
     // More information is in ECMA-130:
     // http://www.ecma-international.org/publications/standards/Ecma-130.htm
-    pub fn generate_subchannel(&self) -> Vec<u8> {
+    pub fn generate_subchannel(
+        &self,
+        chosen_protection_type: &Option<DiscProtection>,)
```

Comment:
Minor nit: the compiler will flag a warning that this param is unused. We can hint that it's fine for now by starting it with an underscore. That way we can safely merge this and follow up in another PR.

```suggestion
        _chosen_protection_type: &Option<DiscProtection>,)
```

Reply:
Sounds good, went and did that.